storybook/webpack.config.ts

```diff
@@ -0,0 +1,31 @@
+import type { Configuration } from "webpack"
+import {
+  elm,
+  excludeExternalModules,
+  babel,
+  styles,
+  svgs,
+  svgIcons,
+  removeSvgFromTest,
+} from "./webpack/rules"
+
+export default ({ config }: { config: Configuration }) => {
+  if (!config.resolve || !config.resolve.extensions || !config.module) {
+    throw new Error(`Storybook started with unexpected config:\n${config}`)
+  }
+
+  // Storybook's base config applies file-loader to svgs
+  config.module.rules = config.module.rules.map(removeSvgFromTest)
+
+  config.module.rules.push(
+    ...[babel, styles].map(excludeExternalModules),
```

Comment:
This is the main thing changed in this config - we only want to run babel and style rules on kaizen/cultureamp/elm storybook packages within node_modules - nothing else. This also fixes the build issues in this PR: https://github.com/cultureamp/kaizen-design-system/pull/1345